Week-1/cattyboard/cattyboard.htm

```diff
@@ -0,0 +1,187 @@
+<html>
+<style>
```

Comment:
Rename to .html

Read about Doctype.

Reply:
Done.
Added Doctype declaration at top.